src/scribe_data/wiktionary/parse_dump.py

```diff
@@ -97,9 +97,10 @@ def _build_iso_mapping(self) -> dict:
         for lang_name, data in language_metadata.items():
             if self.target_iso and lang_name not in self.target_iso:
                 continue
-            iso_code = data.get("iso")
-            if iso_code:
+
+            if iso_code := data.get("iso"):
```

Comment:
There were multiple cases where we can use this "Walrus Operator"/[Assignment Expression](https://peps.python.org/pep-0572/#:%7E:text=known%20as%20%E2%80%9Cthe-,walrus,-operator%E2%80%9D.%20The%20construct%E2%80%99s) to simplify the code :)